homeassistant/components/google_cloud/tts.py

```diff
@@ -107,16 +114,25 @@ def async_get_supported_voices(self, language: str) -> list[Voice] | None:
             return None
         return [Voice(voice, voice) for voice in voices]
 
-    async def async_get_tts_audio(self, message, language, options):
-        """Load TTS from google."""
+    async def async_get_tts_audio(
+        self,
+        message: str,
+        language: str,
+        options: dict[str, Any],
+    ) -> TtsAudioType:
+        """Load TTS from Google Cloud."""
         try:
             options = self._options_schema(options)
         except vol.Invalid as err:
             _LOGGER.error("Error: %s when validating options: %s", err, options)
             return None, None
 
-        encoding = texttospeech.AudioEncoding[options[CONF_ENCODING]]
-        gender = texttospeech.SsmlVoiceGender[options[CONF_GENDER]]
+        encoding: texttospeech.AudioEncoding = texttospeech.AudioEncoding[
+            options[CONF_ENCODING]
+        ]  # type: ignore[misc]
+        gender: texttospeech.SsmlVoiceGender | None = texttospeech.SsmlVoiceGender[
+            options[CONF_GENDER]
+        ]  # type: ignore[misc]
```

Comment:
Where comes the type ignore from?

Reply:
protos? I am just wondering why mypy complains. what's the source type?

Reply:
The error is: `homeassistant/components/google_cloud/tts.py:130: error: The type "type[AudioEncoding]" is not generic and not indexable  [misc]`

The enum is a proto (aka protobuf) enum https://github.com/googleapis/google-cloud-python/blob/957778b3fd25529bbb3f85dac3d9922a50fdaad9/packages/google-cloud-texttospeech/google/cloud/texttospeech_v1/types/cloud_tts.py#L44

Reply:
Oooh, that's what you mean with Protos. My first thought went to javascript prototypes